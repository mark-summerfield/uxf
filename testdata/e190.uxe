uxf 1
[1 2 #<three> 4]
