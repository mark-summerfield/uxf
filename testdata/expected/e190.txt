uxf:E190:e190.uxe:1:inline comments may only occur at the start of Lists, Maps, Tables, and TClasses
